weather_app/management/commands/utils.py

```diff
@@ -0,0 +1,61 @@
+import os
+import csv
+from weather_app.models import WeatherCharacteristic,Weather,CustomTimeStampModel
+from django.core import exceptions
+
+
+""" reading a files form directory"""
+def reading_files(path):
+    all_files = []
+    """
+    loop getting files from path
+    sending files to the "all_files" list
+    """
+    for root, directory, files in os.walk(path):
+        """ here finding .txt entention files """
+        for file in files:
```

Comment:
`file` is keyword in many language so not a good approach to use it as a variable name